### PR TITLE
Rate limit metadata reloads

### PR DESCRIPTION
**Pull request by alena1108**
https://github.com/rancher/rancher/issues/9957

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -17,12 +17,13 @@ import (
 	"syscall"
 	"time"
 
+	_ "net/http/pprof"
```
Comment (ibuildthecloud) [inline main.go]: Are we sure we want this here?

File: main.go [hunk 2]
```diff
@@ -105,6 +106,11 @@ func getCliApp() *cli.App {
 			Name:  "subscribe",
 			Usage: "Subscribe to Rancher events",
 		},
+		cli.Int64Flag{
+			Name:  "reload-interval-limit",
+			Usage: "Limits reload to 1 per interval (milliseconds)",
+			Value: 100,
```
Comment (ibuildthecloud) [inline main.go]: I would assume 100 milliseconds won't make much of a different, what about 1000-2000?

Comment (alena1108) [reply main.go]: I just thought of using this default to be close to the old behavior, but sure, will change it to 1 sec.

File: main.go (hunk 1 above)
Comment (alena1108) [reply main.go]: will remove, leftover from old testing

**Comment by ibuildthecloud** [comment]
LGTM

**Pull request merged**